tests/features/step_definitions/7743.ACs.feature.steps.intTest.ts

```diff
@@ -0,0 +1,34 @@
+import {defineFeature, loadFeature} from 'jest-cucumber';
+import supertest from "supertest";
+import path from 'path';
+
+const url = "http://localhost:3005/";
+const request = supertest(url);
+import mockData from "../../resources/technical-records.json";
+import mockContext from "aws-lambda-mock-context";
+
+const opts = Object.assign({
+  timeout: 0.5
+});
+
+const feature = loadFeature(path.resolve(__dirname, "../7743.ACs.feature"));
+
+defineFeature(feature, test => {
```

Comment:
Does this test not require the DB to be in known state? Probably requires the populateDB/emptyDB beforeAll/afterAll steps

Reply:
Yes, I think it should be pretty easy to implement the beforeAll/afterAll steps